dp-cluster-setup-utility.py

```diff
@@ -1497,7 +1500,21 @@ def update_profiler_agent_service_def(self, current_stack_version):
 
     self._chown_to_knox(service_dir)
     self._chown_to_knox(os.path.dirname(service_dir))
+  
+  def update_ambari_service_def(self, current_stack_version):
```

Comment:
There can be reuse between this and profiler agent copy I think. Please refactor the code appropriately. A lot of code seems to be the same, other than paths of the service files.

Reply:
refactor the code by creating a new  function _execute_service_conf_file_copy_task